nl/mail/chrome/messenger/preferences/applications.dtd

```diff
@@ -24,14 +24,14 @@ entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY enableCloudFileAccountOffer.mb "MB">
-<!ENTITY authRequired.button.label        "Authorize">
+<!ENTITY authRequired.button.label        "Autoriseren">
 <!ENTITY authRequired.button.accesskey    "U">
 <!ENTITY addCloudFileAccount.accesskey    "A">
-<!ENTITY removeCloudFileAccount.label     "Remove">
-<!ENTITY addCloudFileAccount.description  "Add a new online storage service">
-<!ENTITY authRequired.description         "Your authorization is required in order to see the settings for this storage service.">
-<!ENTITY addCloudFileAccount.label        "Add">
+<!ENTITY removeCloudFileAccount.label     "Verwijderen">
+<!ENTITY addCloudFileAccount.description  "Voeg een nieuwe online opslag service toe">
+<!ENTITY authRequired.description         "Uw autotisatie is nodig om de instellingen voor deze opslag service te zien.">
+<!ENTITY addCloudFileAccount.label        "Toevoegen">
 <!ENTITY removeCloudFileAccount.accesskey "R">
-<!ENTITY attachments.incoming.label   "Incoming">
-<!ENTITY attachments.outgoing.label   "Outgoing">
-<!ENTITY enableCloudFileAccountOffer.label "Offer to share for files larger than">
+<!ENTITY attachments.incoming.label   "Inkomend">
+<!ENTITY attachments.outgoing.label   "Uitgaand">
+<!ENTITY enableCloudFileAccountOffer.label "Aanbieden om te delen voor bestanden groter dan">
```

Comment:
CORRECTION MIGHT BE NEEDED, consider: 'Aanbieden om te delen als bestanden groter zijn dan", or 'Bied aan om te delen als bestanden groter zijn dan'' depending on context.
The translation above is technically correct, but it's a literal translation.